Sources/JExtractSwiftLib/Convenience/JavaType+Extensions.swift

```diff
@@ -111,4 +111,29 @@ extension JavaType {
       false
     }
   }
+
+  var wrapperClassIfNeeded: JavaType {
+    switch self {
+    case .boolean:
+      return .class(package: nil, name: "Boolean")
```

Comment:
We probably should have those be properly package-scoped:

```suggestion
      return .class(package: "java.lang", name: "Boolean")
```